linkml_model/model/schema/meta.yaml

```diff
@@ -110,9 +110,22 @@ slots:
     in_subset:
       - owl
 
+  type:
```

Comment:
I suggest `alias_type` OR keeping type generic and putting the specifics in slot_usage... but in this case I think alias_type and alias_type_enum is the way to go

Reply:
I'm not sure what the best approach is here. Could you also have subproperties of `alias_type` such as:
- `has narrow alias`
- `has broad alias`
- `has exact alias`
- `has close alias`
- etc.

From querying standpoint the difference would be:
- Find all alias names where the alias_type = exact
vs.
- Find all values of has exact alias